src/applications/vaos/new-appointment/redux/helpers/formSubmitTransformers.v2.js

```diff
@@ -109,14 +109,20 @@ export function transformFormToVAOSVARequest(state) {
     locationId: data.vaFacility,
     // This may need to change when we get the new service type ids
     serviceType: typeOfCare.idV2,
-    reasonCode: {
-      coding: [
-        {
-          code,
-        },
-      ],
-      text: code,
-    },
+    reasonCode:
+      code === 'Other'
+        ? {
+            coding: [],
+            text: data.reasonAdditionalInfo,
+          }
+        : {
+            coding: [
+              {
+                code,
+              },
+            ],
+            text: code,
```

Comment:
Will this set both the reasonCode coding and text values? On the first read, I interpreted AC number 2 as an XOR.

Reply:
I don't think it matters, I can check Brad Crosby when he returns.  I don't think he mentioned not setting the text field in the case where reason is not Other

Reply:
Looked at the VAOS Service validation code and the changes look good